src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -1260,15 +1264,15 @@ TEST_P(conv_fp32_swish, basic) {
 }
 
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp32_swish, ::testing::ValuesIn(std::vector<convolution_test_params>{
-    // convolution_test_params{ CASE_CONV_FP32_1, 2, 4 },
-    convolution_test_params{ CASE_CONV_FP32_2, 2, 4 },
-    convolution_test_params{ CASE_CONV_FP32_3, 2, 4 },
-    convolution_test_params{ CASE_CONV_FP32_4, 2, 4 },
+    // convolution_test_params{ CASE_CONV_FP32_1, CASE_CONV_FP32_1, 2, 4
```

Comment:
why `CASE_CONV_FP32_1` is repeated? (in other places, too)

Reply:
It was mistake. Done.